### PR TITLE
Accessibility improvements (labels, mnemonics, keyboard navigation)

### PR DESCRIPTION
**Pull request by vvillenave**

### CODE REVIEW

**Review by zhuyaliang — CHANGES_REQUESTED** [review]
```po/fr.po ```Should not appear in this PR

File: trashapplet/src/trashapplet.c [hunk 1]
```diff
@@ -128,12 +131,15 @@ trash_applet_monitor_changed (TrashApplet *applet)
...
           gtk_widget_set_tooltip_text (GTK_WIDGET (applet), text);
+          atk_object_set_description (atk_obj, text);
           g_free (text);
         }
-      else
-        gtk_widget_set_tooltip_text (GTK_WIDGET (applet),
-                                     _("No Items in Trash"));
-
+      else {
+        char *text;
```
Comment (zhuyaliang) [inline trashapplet/src/trashapplet.c:138]: const char *text =  _("No Items in Trash");

**Comment by zhuyaliang** [comment]
@vvillenave How can I test the effect of this PR?

File: trashapplet/src/trashapplet.c (hunk 1 above)
Comment (vvillenave) [reply trashapplet/src/trashapplet.c:138]: I’m not sure I understand what you mean; something like that?
```
  if (items != applet->items)
    {
      const char *text = _("No Items in Trash");
      if (items)
        {
          text = g_strdup_printf (ngettext ("%d Item in Trash",
                                            "%d Items in Trash",
                                            items), items);
        }
      gtk_widget_set_tooltip_text (GTK_WIDGET (applet), text);
      g_free (text);

      applet->items = items;
    }
```

Comment (zhuyaliang) [reply trashapplet/src/trashapplet.c:138]:
```
const char *text =  _("No Items in Trash");
gtk_widget_set_tooltip_text (GTK_WIDGET (applet), text);
atk_object_set_description (atk_obj, text);
```

Comment (zhuyaliang) [reply trashapplet/src/trashapplet.c]: I don't think it's necessary to allocate memory for string constants at all

**Review by rbuj — APPROVED** [review]
Right click on the sticky note title shows the popup menu which works properly.

Captura de Pantalla 2021-10-20 a les 17 16 41


I didn't check the other accessibility features.

**Comment by raveit65** [comment]
Here we go, thx.
https://github.com/mate-desktop/mate-applets/commits/1.26

**Pull request merged**